Makefile

```diff
@@ -37,21 +37,18 @@ BUILD_JOBS := $(addprefix build-,${ENVS})
 PUBLISH_JOBS := $(addprefix publish-,${ENVS})
 
 .PHONY: all
-all: build ## Prepare and run the project tests
+all: test build ## Prepare and run the project tests
 
 .PHONY: install
-install: publish-local ## Install the plugin binaries to the local machine
+install: test## Install the plugin binaries to the local machine
 	@# TODO avoid deleting an existing plugin once in place reinstalls are working again
 	@tanzu plugin delete apps > /dev/null 2>&1 || true
-	tanzu plugin install apps --version $(BUILD_VERSION) --local $(TANZU_PLUGIN_PUBLISH_PATH)/${GOHOSTOS}-${GOHOSTARCH}
-
-.PHONY: build-local
-build-local: prepare test ## Build the plugin binaries locally for the host architecture
-	echo BUILD_VERSION: $(BUILD_VERSION)
 	tanzu builder cli compile --version $(BUILD_VERSION) --ldflags "$(LD_FLAGS)" --path ./cmd/plugin --target local --artifacts ${ARTIFACTS_DIR}/${GOHOSTOS}/${GOHOSTARCH}/cli
+	tanzu builder publish --type local --plugins "apps" --version $(BUILD_VERSION) --os-arch "${GOHOSTOS}-${GOHOSTARCH}" --local-output-discovery-dir "$(TANZU_PLUGIN_PUBLISH_PATH)/${GOHOSTOS}-${GOHOSTARCH}/discovery/standalone" --local-output-distribution-dir "$(TANZU_PLUGIN_PUBLISH_PATH)/${GOHOSTOS}-${GOHOSTARCH}/distribution" --input-artifact-dir $(ARTIFACTS_DIR)
```

Comment:
is it possible here to call `publish-${GOHOSTOS}-${GOHOSTARCH}`? instead writing the command again?

Reply:
We can call the `publish-${GOHOSTOS}-${GOHOSTARCH}` however it would create a tar file that is not necessary for install. We wanted to avoid creating a tar while calling make install.